uk.co.compucorp.civicrm.hrleaveandabsences/js/src/leave-absences/leave-type-wizard/form/components/leave-type-wizard-form.html

```diff
@@ -1,3 +1,24 @@
 <div>
   <h1>Create leave/absence type</h1>
+
+  <div class="panel-group">
+    <div class="panel panel-default"
+      ng-repeat="section in form.sections">
+      <div class="panel-heading"
+        ng-class="{ pointer: !section.active }"
+        ng-click="form.openSection(section.name)">
+        <h4 class="panel-title">
+          <small>
+            <i class="fa fa-chevron-{{section.active ? 'down' : 'right'}}"></i>
```

Comment:
i know the UI is not finalized, but FYI this is not the standard markup we have for collapsible panels (check the styleguide)

Reply:
Done.